### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Application_Keyboard : Inactive elements such as heading text receives keyboard focus on navigation.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3500]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG2.1.1;#MAC;#A11ySev2;#Benchmark;#Keyboard;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to "Application" on right pane and click on it.
4. "Application" screen gets opened.
5. Using keyboard navigate though the "Application" screen and observe if the heading text receives keyboard focus or not.

### Actual:
When keyboard users navigate through the screen content the inactive element such as heading text present on the screen i.e. the "Application" heading text receives keyboard focus which is incorrect.

This issue is observed in the below scenarios:
1. Settings - Data Explorers
2. Settings - Transfers

### Expected:
Inactive element such as heading text should not receive keyboard focus while navigation using keyboard.

### User Impact:
If inactive element receive keyboard focus for the content present on the screen then the keyboard only user will interpret them as actionable control and while try activating them. This will lead to confusion as incorrect information is conveyed to them.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EW45SLwiT8lHuucd9_028dsBBTxyztkoksyxmOUOddC4ww?e=IpYgm0

**Comment by JasonYeMSFT** [comment]
@MayuriNaik Could you please verify and close this issue if it is fixed?

**Comment by MayuriNaik** [comment]
For keyboard users the heading text now does not receive keyboard focus while navigation using 'Tab' key. The bug is fixed and hence closing the bug.Verified in application version Version: 1.16.0-dev AzCopy Version: 10.6.0.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #Closed
